spec/features/orders/user_orders_index_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "A user sees thier completed orderss" do
+  scenario "logged-in user can see all completed orders" do
+
+    user = create(:user)
+    user_order1 = create(:order, user_id: user.id)
+    user_order2 = create(:order, user_id: user.id)
+
```

Comment:
yeah, that's a really good point. I need to get some paths in there to test for an unauthenticated user trying to look at someone else's orders.